.gitignore

```diff
@@ -360,3 +360,4 @@ healthchecksdb
 
 # Backup folder for Package Reference Convert tool in Visual Studio 2017
 MigrationBackup/
+Source/Documentation/Manual/.vscode/settings.json
```

Comment:
A personal preference snuck in here

Reply:
Thanks, Ryan

I've removed that now.

Best wishes,

Chris